fenix/app/src/main/java/org/mozilla/fenix/compose/ImagesPlaceholder.kt

```diff
@@ -21,61 +25,81 @@ import org.mozilla.fenix.theme.FirefoxTheme
 /**
  * Renders the app default image placeholder while the image is still getting loaded.
  *
+ * @param placeholder [Painter] placeholder painter used during loading. By default, a solid color is used.
  * @param modifier [Modifier] allowing to control among others the dimensions and shape of the image.
  * @param contentDescription Text provided to accessibility services to describe what this image represents.
  * Defaults to [null] suited for an image used only for decorative purposes and not to be read by
  * accessibility services.
  */
 @Composable
 internal fun ImageLoaderScope.WithDefaultPlaceholder(
+    placeholder: Painter,
     modifier: Modifier,
     contentDescription: String? = null,
 ) {
     Placeholder {
-        DefaultImagePlaceholder(modifier, contentDescription)
+        DefaultImagePlaceholder(placeholder, modifier, contentDescription)
     }
 }
 
 /**
  * Renders the app default image placeholder if loading the image failed.
  *
+ * @param placeholder [Painter] placeholder painter used if loading failed. By default, a solid color is used.
  * @param modifier [Modifier] allowing to control among others the dimensions and shape of the image.
  * @param contentDescription Text provided to accessibility services to describe what this image represents.
  * Defaults to [null] suited for an image used only for decorative purposes and not to be read by
  * accessibility services.
  */
 @Composable
 internal fun ImageLoaderScope.WithDefaultFallback(
+    placeholder: Painter,
     modifier: Modifier,
     contentDescription: String? = null,
 ) {
     Fallback {
-        DefaultImagePlaceholder(modifier, contentDescription)
+        DefaultImagePlaceholder(placeholder, modifier, contentDescription)
     }
 }
 
 /**
  * Application default image placeholder.
  *
+ * @param placeholder [Painter] placeholder provided when there is no resource. By default, a solid color is used.
  * @param modifier [Modifier] allowing to control among others the dimensions and shape of the image.
  * @param contentDescription Text provided to accessibility services to describe what this image represents.
  * Defaults to [null] suited for an image used only for decorative purposes and not to be read by
  * accessibility services.
  */
 @Composable
 internal fun DefaultImagePlaceholder(
+    placeholder: Painter,
     modifier: Modifier,
     contentDescription: String? = null,
 ) {
-    Image(ColorPainter(FirefoxTheme.colors.layer2), contentDescription, modifier)
+    Box(
+        modifier = Modifier
+            .background(
+                color = FirefoxTheme.colors.layer3,
+                shape = RoundedCornerShape(8.dp),
+            ),
+    ) {
```

Comment:
Not sure this should be in the default image placeholder, seems like the intention was to just have a color?